src/Microsoft.Data.Sqlite/SqliteDataReader.cs

```diff
@@ -30,9 +29,10 @@ public class SqliteDataReader : DbDataReader
         private bool _closed;
 
         internal SqliteDataReader(
-            Sqlite3Handle db,
+            SqliteConnection connection,
             Queue<Tuple<Sqlite3StmtHandle, bool>> stmtQueue,
-            int recordsAffected)
+            int recordsAffected,            
+            CommandBehavior behavior)
```

Comment:
Just make this `bool closeConnection`; this class doesn't care about the other values.